README.md

```diff
@@ -1562,3 +1562,12 @@ Projects in C/C++ ([go to C#](#csharp), [go to Java](#java))
 	- XMage is an open-source collectible card game.
 	- http://xmage.de/
 	- https://github.com/magefree/mage
+
+- MSVC STL
+        - The implementation of the C++ Standard Library for Visual Studio
+        - https://github.com/microsoft/STL
+
+- LLVM 
+        - The LLVM Compiler Infrastructure
+        - https://llvm.org/
+        - https://github.com/llvm/llvm-project
```

Comment:
We've already checked LLVM project twice: [LLVM 11](https://pvs-studio.com/en/blog/posts/cpp/0771/) and [LLVM 13](https://pvs-studio.com/en/blog/posts/cpp/0871/). There is no need to add this project as we track LLVM releases and check the latest versions.

Please, remove this from your PR.

Reply:
Done. And thanks for direct links, reading these now.